src/sotoki/constants.py

```diff
@@ -48,6 +48,13 @@
 
 
 def lang_for_domain(domain):
+    """
```

Comment:
Doesn't respect our code formatting (black): single summary on same line then a blank one then the rest of the docstring